src/Codeception/Attribute/Depends.php

```diff
@@ -1,10 +1,15 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Codeception\Attribute;
 
 use Attribute;
 
 #[Attribute(Attribute::TARGET_CLASS | Attribute::TARGET_METHOD | Attribute::IS_REPEATABLE)]
 final class Depends
 {
+    public function __construct(string $testName)
```

Comment:
`Depends` is repeatable so the constructor params can be changed to `string ...$testNames`

Reply:
Digging through the code it seems that depends should be flattened as well. I'm not even sure the current code works properly for Depends. I'll have to test this a bit.

Reply:
Sorry, yep, repeatable was a bad way for me to describe it. I should've said it can take multiple parameters instead. You're correct in that there's no tests which cover multiple test names to the `Depends` attribute, however manually testing this I confirmed that it could take multiple test names and correctly act accordingly to whether those tests passed or failed. If I get time today/tonight I'll write a test for multiple methods in `Depends` and add the patch for it here.

Reply:
I'll change it like you suggested then

Reply:
Thanks. I've added the change to the MR 👍